crop.go

```diff
@@ -433,24 +414,19 @@ func edgeDetect(i image.Image, o image.Image) {
 
 func faceDetect(settings CropSettings, i image.Image, o image.Image) error {
 
-	cvImage := opencv.FromImage(i)
 	_, err := os.Stat(settings.FaceDetectionHaarCascadeFilepath)
 	if err != nil {
 		return err
 	}
 	cascade := opencv.LoadHaarClassifierCascade(settings.FaceDetectionHaarCascadeFilepath)
+	defer cascade.Release()
+	cvImage := opencv.FromImage(i)
+	defer cvImage.Release()
 	faces := cascade.DetectObjects(cvImage)
 
 	gc := draw2dimg.NewGraphicContext((o).(*image.RGBA))
 
-	if settings.DebugMode == true {
```

Comment:
I'd actually still like to keep debug output for such things. Maybe we could just offer a logger interface. that the user can set, and print to it if it's not nil.